next.config.js

```diff
@@ -10,7 +10,12 @@ const withSourceMaps = require('@zeit/next-source-maps')({
 })
 
 const basePath = process.env.NEXT_PUBLIC_URLS_BASE_PATH || ''
-
+const url = process.env.VERCEL_URL || 'http://localhost:3001/'
+const devAssetsRegex = 'https://prod-tab2017-media.gladly.io/.*'
+const prodAssetsRegex = 'https://dev-tab2017-media.gladly.io/.*'
+const cachingRegex = new RegExp(
```

Comment:
We'll need this to work for our dev/prod domains, too. Let's just include the following, hardcoded:
`https://dev-tab2017.gladly.io/newtab/.*`
`https://tab.gladly.io/newtab/.*`

You can test by merging into the `dev` branch and making sure all works as expected [on dev](https://dev-tab2017.gladly.io/newtab/).

Reply:
@kmjennison I believe that the VERCEL_URL will resolve to those values at each of those environments

Reply:
testing now on dev